moodlecli/main.py

```diff
@@ -287,3 +287,10 @@ def export_bulk_csv(output_csv):
         utils.bulk_export_csv_course_ids()
     )
     writer.writeheader()
+
+
+@cli.command()
+@click.argument('content_id')
+def log_event_manually(content_id):
+    moodle = get_moodle_client()
```

Comment:
In so much as this is a helper command to test the APIs for the spike, let's create a single command `trigger-contentloaded-event` and have it take a parameter like `type` which would be a `click.Choice`. In this PR we can just allow a single option of `direct`, but then later we can add `moodle`. That way it's set up for us to use it to test both endpoints for k12-182 and k12-183.

Reply:
Oh that's handy. Sounds good 👍

Reply:
Ah, sorry, I may have created some confusion here in my initial comment. I think including the `content_id` as an input is still a good idea. I think you may have made a change to hard code that now to `test_id`. Similarly, I think you had the better call signature for the moodle client function previously as the convention is we create the data structure expected by the service there versus in this CLI handler.